Presentation.Web/Models/External/V2/Response/SystemUsage/GDPRRegistrationsResponseDTO.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using Presentation.Web.Models.External.V2.Types.Shared;
+using Presentation.Web.Models.External.V2.Types.SystemUsage;
+
+namespace Presentation.Web.Models.External.V2.Response.SystemUsage
+{
+    public class GDPRRegistrationsResponseDTO
+    {
+        public string Purpose { get; set; }
```

Comment:
Nogen grund til at der ikke er samme beskrivelse i denne DTO som i de andre?

Reply:
Jeg holdt op med at tilføje beskrivelser til felter der ikke krævede ekstra forklaring... f.eks. Description er heller ikke beskrevet. "Purpose" tænker jeg heller ikke kræver forklaringen "The purpose of the IT-System" :-)

Reply:
Jeg gravede lidt i om det kunne være "Formålet med dataindsamlingen" men det lader ikke til at være tilfældet